### DISCUSSION TITLE
Has anyone tried using this on Replit?

### DISCUSSION THREAD

**Discussion by jbistro** [AntonOsika/gpt-engineer#108] [Q&A]
I am running into an issue where it can not find the module and I am not sure if this is a replit issue or something else.  Thanks!

~/gpt-engineer/gpt_engineer$ python3 main.py uptime_app
Traceback (most recent call last):
  File "/home/runner/gpt-engineer/gpt_engineer/main.py", line 6, in <module>
    from gpt_engineer.chat_to_files import to_files
ModuleNotFoundError: No module named 'gpt_engineer'

**Comment by patillacode** [comment]
Hi @jbistro 

there were some changes added to the way we execute the application, please pull the latest `main` version and follow the README file.

Once you have updated the code you will need to call the application like so:
`python -m gpt_engineer.main uptime-app`
